tests/std/tests/P0355R7_calendars_and_time_zones_io/test.cpp

```diff
@@ -888,22 +892,68 @@ void insert_leap_second(const sys_days& date, const seconds& value) {
         tzdb{my_tzdb.version, move(zones), move(links), move(leap_vec), my_tzdb._All_ls_positive && (value == 1s)});
 }
 
+void test_gh_1952() {
+    auto time_str{"2021-06-02T17:51:05.696028Z"};
+    auto fmt{"%FT%TZ"};
+    auto utc_ref = clock_cast<utc_clock>(sys_days{2021y / June / 2d} + 17h + 51min + 05s + 696028us);

```

Comment:
These could be top-level `const`. We're not super-rigorous about this, especially for short/simple functions in test code, but in longer/complicated test functions (this one is moderately so), it helps to indicate that we won't be mutating these variables, avoids accidental mutation, and (when done consistently) the lack of `const` on other variables indicates that we *will* be mutating them.